lang/en/subcourse.php

```diff
@@ -71,6 +71,8 @@
 $string['settings:coursepageprintgrade_desc'] = 'Display grade from referenced course on course page.';
 $string['settings:coursepageprintprogress'] = 'Progress on course page';
 $string['settings:coursepageprintprogress_desc'] = 'Display progress from referenced course on course page.';
+$string['settings:displayhiddencourses'] = 'Display Hidden Courses';
```

Comment:
Moodle uses sentence case - not camel case - can you please change to "Display hidden courses"